pallets/runtime/common/tests/all/group_test.rs

```diff
@@ -42,14 +42,14 @@ fn add_member_works() {
 }
 
 fn add_member_works_we() {
-    let root = Origin::system(frame_system::RawOrigin::Root);
+    let root = Origin::from(frame_system::RawOrigin::Root);
     let non_root = Origin::signed(AccountKeyring::Bob.public());
     let non_root_did = get_identity_id(AccountKeyring::Alice).unwrap();
 
     Context::set_current_identity::<Identity>(Some(non_root_did));
-    assert_noop!(
-        CommitteeGroup::add_member(non_root, IdentityId::from(3)),
-        group::Error::<TestStorage, group::Instance1>::BadOrigin
+    assert_eq!(
```

Comment:
The previous check is different from the new one. 
Previously we were double-check explicitly for a `BadOrigin`, and the new one is evaluating any error.
There are some similar cases below.

Reply:
Resolved